### ISSUE TITLE
Add Faroe Islands installed generation capacity

### ISSUE THREAD

**Issue by alixunderplatz** [tmrowco/electricitymap#612]
Hi everyone,

I could find some useful information for the yet undefined installed capacities on the **Faroe Islands**.
There is [a thesis with a list of the generation capacities (p. 44)](https://setur.fo/uploads/tx_userpubrep/BScThesis_JohanPauliMagnussen.pdf), which are as follows:

- 18.3 MW wind power
- 39.4 MW hydro power and
- 66.7 MW for fossil generation capacity (oil)

For the moment, these numbers should be valid.

Greetings, Alex

**Comment by corradio** [comment]
Hi @alixunderplatz ,

Thanks for the info. Are you able to submit a pull request?

Olivier

**Comment by alixunderplatz** [comment]
Hi @corradio ,

as I'm only getting started to understand how all of this works on the programming side, I'm not too confident of submitting a pull request.
However, I have managed to fork the repository and added the mentioned capacities to capacities.json and updated sources to the code in readme.md. 
But I did not yet manage to properly use docker to check if everything is running as desired.

Should I submit the pull requests anyway?

Alex
